test/integration/client-side-operations-timeout/client_side_operations_timeout.spec.test.ts

```diff
@@ -1,9 +1,61 @@
 import { join } from 'path';
+import * as semver from 'semver';
 
 import { loadSpecTests } from '../../spec';
 import { runUnifiedSuite } from '../../tools/unified-spec-runner/runner';
 
-// TODO(NODE-5823): Implement unified runner operations and options support for CSOT
-describe.skip('CSOT spec tests', function () {
-  runUnifiedSuite(loadSpecTests(join('client-side-operations-timeout')));
+const skippedSpecs = {};
+
+const skippedTests = {
+  'Tailable cursor iteration timeoutMS is refreshed for getMore - failure': 'TODO(DRIVERS-2965)',
+  'Tailable cursor awaitData iteration timeoutMS is refreshed for getMore - failure':
+    'TODO(DRIVERS-2965)',
+  'command is not sent if RTT is greater than timeoutMS': 'TODO(DRIVERS-2965)',
+  'Non-tailable cursor iteration timeoutMS is refreshed for getMore if timeoutMode is iteration - failure':
+    'TODO(DRIVERS-2965)',
+  'maxTimeMS value in the command is less than timeoutMS':
+    'TODO(DRIVERS-2970): see modified test in unified-csot-node-specs',
+  'timeoutMS is refreshed for getMore - failure':
+    'TODO(DRIVERS-2965): see modified test in unified-csot-node-specs', // Skipping for both tailable awaitData and tailable non-awaitData cursors
```

Comment:
confused about this comment

Reply:
Moving discussion to https://github.com/mongodb/node-mongodb-native/pull/4315/files